src/fsharp/FSharp.Core/prim-types.fs

```diff
@@ -650,23 +663,23 @@ namespace Microsoft.FSharp.Core
     open Microsoft.FSharp.Core.BasicInlinedOperations
 
     //-------------------------------------------------------------------------
-    // The main aim here is to bootsrap the definition of structural hashing 
+    // The main aim here is to bootstrap the definition of structural hashing 
     // and comparison.  Calls to these form part of the auto-generated 
     // code for each new datatype.
 
     module LanguagePrimitives =  
    
-        [<Sealed>]
-        type (* internal *) ErrorStrings =
+
+        module (* internal *) ErrorStrings =
```

Comment:
Do I understand correctly that this is now only in place for backward compatibility?